.github/workflows/sync_with_container_templates_bucket.yml

```diff
@@ -0,0 +1,35 @@
+name: Sync with container templates S3 bucket
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  zip_file_name: ${{ github.event.repository.name }}-${{ github.head_ref || github.ref_name }}.zip
```

Comment:
In the future we would have these actions in a shared repository (e.g. newscred/github-actions) and references them here.